configs/13.0/packages/expat/sources

```diff
@@ -21,7 +21,7 @@
 
 ATSRC_PACKAGE_NAME="Expat XML Parser"
 ATSRC_PACKAGE_VER="2.2.6"
-ATSRC_PACKAGE_REV=5801a1b70ade
+ATSRC_PACKAGE_REV=39e487da353b
```

Comment:
The version and revision match and look good for this pull request:

```
$ git describe 39e487da353b
R_2_2_6
```

On the other hand, is there a reason to not update it the more recent tag, i.e.:

```
$ git describe d3b78b42a2dc
R_2_2_7
```